frontend/src/Views/AccountCreation/AccountCreation.tsx

```diff
@@ -29,25 +30,45 @@ const useStyles = makeStyles((theme: Theme) => ({
     }
   }
 }));
-
-const AccountCreation = () => {
+interface IProps {
+  onSubmit?: (account: IAccount) => void;
```

Comment:
Kam sitas reikalingas? `AccountCreation` yra top-level component, jam praktiskai nera kas galetu paduot props?

Reply:
turėt tokį optional onSubmit callback'ą būtų visai naudinga page, kuris naudoja accountus, nes kai įvyksta onSubmit, žinai, kad reik paupdate'int duomenis

Reply:
Tikrai taip, jei assuminam, kad mes turim global state (store), kuriame patys managiname app state, be backendo. Bet kadangi buvo nuspresta praeita karta nedaryti redux, tai nelabai naudingas siuo atveju. + `AccountCreation`, velgi, yra top-level component, tai kas jam duos onSubmit callbacka ir kur tas state bus laikomas? Jei `App`e tai griztama prie tos pacios redux problemos :D
Ir dar ++ net jei reiketu kazka globaliau pakeisti tai tai turetu daryti pats `AccountCreation` component, o ne issiuntineti notificationus i X skirtingu vietu ir duplikuoti account state updatinimo logika :D Siai problemai spresti grynai ir yra skirtas redux

Reply:
Nu ok tai trink šitą